service/history/replication/quotas.go

```diff
@@ -0,0 +1,24 @@
+package replication
+
+import (
+	"go.temporal.io/server/common/quotas"
+)
+
+const (
+	taskSchedulerToken = 1
+)
+
+type (
+	ServerSchedulerRateLimiter quotas.RequestRateLimiter
+	ClientSchedulerRateLimiter quotas.RequestRateLimiter
+)
```

Comment:
are they used now?

Reply:
They are initiated as no op rate limiter in the fx provider.